lib/components/package.json

```diff
@@ -37,7 +37,7 @@
     "*.d.ts"
   ],
   "scripts": {
-    "prepare": "ts-node ../../scripts/prebundle.ts"
+    "prepare": "../../node_modules/.bin/ts-node ../../scripts/prepare-prebundle.ts"
```

Comment:
I'm with @yannbf on this one -- I think this is a bad pattern since it ties our packages to the monorepo

Reply:
The package IS in the monorepo?
This is a script only ever run from within the monorepo?

How would `"prepare": "ts-node ../../scripts/prebundle.ts"` outside of the monorepo?